gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/ServiceModelGeneratorHandleResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.knox.gateway.topology.discovery.cm;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class ServiceModelGeneratorHandleResponse {
+  private final Collection<String> configurationIssues = new HashSet<>();
+  private boolean handles;
+
+  public ServiceModelGeneratorHandleResponse(boolean handles) {
+    this.handles = handles;
+  }
+
+  public boolean isHandles() {
```

Comment:
`isHandles` grammatically sounds wrong. `wasHandled` or something along those lines. Maybe hasError?

Reply:
`hasError` does not really express what we are doing here. IT might happen that it's not handled (e.g. role type is different) but it does not mean it has errors.
What about `public boolean handled()`?

Reply:
yea basically just not `isHandles` :)

Reply:
Fixed